src/index.html

```diff
@@ -2,15 +2,49 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no,
+     initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+     >
```

Comment:
do not format it, previous formatting was good 
![image](https://user-images.githubusercontent.com/64437769/142187650-d2637499-235d-40ec-94ac-108cb07e38eb.png)